packages/react/src/List/listContext.ts

```diff
@@ -0,0 +1,5 @@
+import {createContext} from 'react'
+
+export const ListContext = createContext({
+  variant: 'default'
```

Comment:
This is shared with OrderedList, so should it be default? Initialising as default makes sense for Unordered though.

Reply:
It should be initialized to something picking ordered or unordered list to make default doesn't really mean much to me. And it functionally acts the same way as the state is handled internally.